src/core/registry.ts

```diff
@@ -1,7 +1,7 @@
 import { extend } from '../shared/language-util';
 import { forEach, kebabToCamelCase } from '../shared/util';
 import type { ComponentProto, Grammar } from '../types';
-import type { Prism } from './prism';
+import type { Prism } from './prism-class';
 
```

Comment:
```suggestion
import type Prism from './prism-class';
```

Another option would be:

```ts
import type { Prism } from './prism';
```

Reply:
We want to import from the lowest level module we can